src/registrar/templates/includes/request_review_steps.html

```diff
@@ -132,7 +132,7 @@ <h3 class="register-form-review-header">Alternative domains</h3>
         {% with title=form_titles|get_item:step %}
             {% if domain_request.has_additional_details %}
                 {% include "includes/summary_item.html" with title="Additional Details" value=" " heading_level=heading_level editable=is_editable edit_link=domain_request_url %}
-                <h3 class="register-form-review-header">CISA Regional Representative</h3>
+                <h3 class="header--body text-primary-dark margin-bottom-0">CISA Regional Representative</h3>
```

Comment:
**suggestion:** there are 10 places using this same css in this PR alone, how about instead we either update header--body or just make a new css definition that extends header--body and include the text dark and 0 bottom margin customizations. That way if we want to change this down the road we aren't changing it in 10 places

Reply:
See my comment above. Design introduced a new header variant, so we now have 2 types of headers that share the same CSS (size, top margin) but differ on bottom margin and color. I believe the common CSS should be defined in the common class (header--body) and the variations on bottom margin and color should either be defined in a secondary class or be applied through the utility classes as done here.

Ideally - and this should be posed to @cisagov/gov-designers - we do not need such minor variations on color and margins and we can get rid of the utility classes + rename header--body or just apply the styles of header--body directly to `h4` and a class called `.h4 `

Reply:
Agree that small variations should be avoided. cc: @gabydisarli for input

Reply:
I think @Katherine-Osos and @gabydisarli this really applies to the new ticket for auditing all of our styles/ headings and creating a more consistent solution. Definitely counting this out of scope for this PR.